docs/wiki/deployment/process-auditing.md

```diff
@@ -1,25 +1,28 @@
 
-Osquery has the ability to leverage the Linux Audit System to record process executions and network connections in near real-time on Linux and MacOS systems. Although these auditing features are extremely powerful for recording the activity from a host, they may introduce additional computational overhead and greatly increase the number of log events generated by osquery.
+osquery can leverage the audit subsystems to record process executions and network connections in near real-time on Linux and MacOS systems. Although these auditing features are extremely powerful for recording the activity from a host, they may introduce additional computational overhead and greatly increase the number of log events generated by osquery.
 
-Enabling these auditing features requires additional configuration to osquery. The `process_events` and `socket_events` tables use the same event-based architecture as the [File Integrity Monitoring (FIM)](../deployment/file-integrity-monitoring.md). To read more about how event-based tables are created and designed, check out the osquery [Table Pubsub Framework](../development/pubsub-framework.md). On all supported platforms, process events are abstracted into the  [`process_events`](https://osquery.io/schema/2.11.2#process_events) table. Similarly, socket events are abstracted into the [`socket_events`](https://osquery.io/schema/2.11.2#process_events) table.
+Enabling these auditing features requires additional configuration of osquery. The `process_events` and `socket_events` tables use the same event-based architecture as the [File Integrity Monitoring (FIM)](../deployment/file-integrity-monitoring.md). To read more about how event-based tables are created and designed, check out the osquery [Table Pubsub Framework](../development/pubsub-framework.md). On all supported platforms, process events are captured in the  [`process_events`](https://osquery.io/schema/2.11.2#process_events) table. Similarly, socket events are captured in the [`socket_events`](https://osquery.io/schema/2.11.2#process_events) table.
 
 To collect process events add a query like:
+
 ```sql
 SELECT * FROM process_events;
 ```
+
 to your query schedule, or to a query pack.
 
 ## Linux process auditing
 
-osquery uses the Linux Audit System to collect and process audit events from the kernel. It accomplishes this by monitoring the `execve()` syscall. Auditd should not be running when using osquery's process auditing, as it will conflict with osqueryd over access to the audit netlink socket. You should also ensure auditd is not configured to start at boot.
+On Linux, osquery uses the Audit system to collect and process audit events from the kernel. It accomplishes this by monitoring the `execve()` syscall. `Auditd` should not be running when using osquery's process auditing, as it will conflict with `osqueryd` over access to the audit netlink socket. You should also ensure `auditd` is not configured to start at boot.
```

Comment:
```suggestion
On Linux, osquery uses the Audit system to collect and process audit events from the kernel. It accomplishes this by monitoring the `execve()` syscall. `auditd` should not be running when using osquery's process auditing, as it will conflict with `osqueryd` over access to the audit netlink socket. You should also ensure `auditd` is not configured to start at boot.
```